tests/expected-schema.json

```diff
@@ -65,7 +65,7 @@
                 },
                 "tag": {
                     "type": "string",
-                    "description": "Kubewatch image tag",
+                    "description": "",
                     "default": "0.1.0-debian-10-r162"
                 },
```

Comment:
Shouldn't it be ignored also in the schema? Also the default value stablished in the modifier should apply to the  schema also.